variegated-embassy-ads124s08/src/registers.rs

```diff
@@ -942,9 +1332,16 @@ impl ByteRepresentation for ReferenceControlRegister {
 }
 
 #[derive(Debug, Clone, Copy, Eq, PartialEq, Default, Format)]
+/// IDACMagnitudeRegister
```

Comment:
This enum can be better explained, reference the data sheet to craft a better explanation.